hraven-etl/src/main/java/com/twitter/hraven/mapreduce/GraphiteHistoryWriter.java

```diff
@@ -212,7 +216,7 @@ private String getMetricsPath() throws IOException {
       
       StandardEvaluationContext context = new StandardEvaluationContext();
       Map<String, Object> systemTokens = getSystemTokens();
-      Map<String, String> userTokens = getUserTags();
+      userTokens = getUserTags();
```

Comment:
if you're making this a class variable, don't initialize it here. initialize it in the constructor.